docs/google/engine.md

```diff
@@ -1,9 +1,17 @@
 # Quantum Engine
 
-The Quantum Engine, via the `cirq.google.Engine` class, executes programs and jobs using the
-Quantum Engine API.
-
-Note that the Quantum Engine API is not yet open for public access.
+Google's Quantum Computing Service provides the Quantum Engine API to execute 
+circuits on Google's quantum processor or simulator backends and 
+to access or manage the jobs, programs, reservations and calibrations. Cirq is 
+the only supported client for this API, using the `cirq.google.Engine` class. 
+
+Note: the Quantum Engine API is not yet open for public access.
+
+Note: while Cirq is the only supported way of using to the API currently, 
```

Comment:
nit:  It's slightly awkward to have two "Note:" in a row.  Maybe move this note up to the previous paragraph and condense it, such as:

Cirq is the only supported client for this API, using the `cirq.google.Engine` class.  For other use cases (e.g. from a different language), contact [cirq-maintainers@googlegroups.com](mailto:cirq-maintainers@googlegroups.com) with a proposal.